src/util.lisp

```diff
@@ -81,7 +78,14 @@
           ((vector (unsigned-byte 8))
            (length body))))))
 
+;; cl-isaac supports ISAAC-64 solely for implementations with x86-64
+;; capabilities. Use whichever-best supported capability
+(defparameter *isaac-ctx*
+  (isaac:init-self-seed :count 5
+                        :is64 #+:X86-64 t #-:X86-64 nil))
+
 (defun generate-random-id ()
   "Generates a random token."
-  (ironclad:byte-array-to-hex-string
-    (ironclad:random-data 20)))
+  (format nil "~64,'0x" (#+:X86-64 isaac:rand-bits-64
+                         #-:X86-64 isaac:rand-bits
+                         *isaac-ctx* 512)))
```

Comment:
I found another problem.
This generates a 128-char string, though the previous version generates 40-chars.

It could break some compatibility.
For example, the following is a validation code in lack.middleware.session that assumes the session ID is a lower-cased 40-char string.

https://github.com/fukamachi/lack/blob/156ef010d5a90e629107b44661221b26b507d7ef/src/middleware/session/state.lisp#L18-L22

Reply:
If it's okay with you, this snippet works for me.

```suggestion
  (format nil "~(~40,'0x~)" (#+:X86-64 isaac:rand-bits-64
                             #-:X86-64 isaac:rand-bits
                             *isaac-ctx* 160)))
```

Reply:
LGTM, applied